client/web/compose/src/components/PageBlocks/GeometryBase.vue

```diff
@@ -14,76 +14,48 @@
     <div
       v-else
       class="w-100 h-100"
-      @mouseover="disableMap"
-      @mouseleave="enableMap"
     >
-      <l-map
-        v-if="map"
-        ref="map"
-        :zoom="map.zoom"
-        :center="map.center"
-        :min-zoom="map.zoomMin"
-        :max-zoom="map.zoomMax"
-        :bounds="map.bounds"
-        :max-bounds="map.bounds"
+      <c-map
+        ref="c-map"
+        :map="{
+          ...map,
+          maxBounds: map.bounds
+        }"
+        :labels="{
+          tooltip: { 'goToCurrentLocation': $t('geometry.tooltip.goToCurrentLocation') }
+        }"
+        :markers="localValue"
+        :disable="editable"
+        :hide-geo-search="true"
```

Comment:
you dont need to say `hide-geo-search="true"`. If you just do `hide-geo-search` it will be true